### PR TITLE
Upgrades packages

### PR DESCRIPTION
**Pull request by 2roy999**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+  - yarn install
+
+# Post-install test scripts.
+test_script:
+  # run tests
+  - yarn lint
+  - yarn test
+
+# Don't actually build.
+build: false
```
Comment (idanstark42) [inline appveyor.yml]: How come? It works anyway, and it's using `off` in all our other packages

File: README.md [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
-| clientId| An ID for the client | Random base64 text |
-| reconnectTimeout | Time between connection attempts | 10 seconds |
-| logger | A logger object of the pattern `{ debug, info, warn, error, fatal }`, can axcept `ms-logger` | empty logger |
+[![npm](https://img.shields.io/npm/v/@first-lego-league/ms-messenger.svg)](https://www.npmjs.com/package/@first-lego-league/ms-messenger)
+[![David Dependency Status](https://david-dm.org/FirstLegoLeague/ms-messenger.svg)](https://david-dm.org/FirstLegoLeague/ms-messenger)
+[![David Dev Dependency Status](https://david-dm.org/FirstLegoLeague/ms-messenger/dev-status.svg)](https://david-dm.org/FirstLegoLeague/ms-messenger#info=devDependencies)
+[![David Peer Dependencies Status](https://david-dm.org/FirstLegoLeague/ms-messenger/peer-status.svg)](https://david-dm.org/FirstLegoLeague/ms-messenger?type=peer)
+[![Build status](https://ci.appveyor.com/api/projects/status/0ya8dl62755nn01g/branch/master?svg=true)](https://ci.appveyor.com/project/2roy999/ms-messenger/branch/master)
+[![GitHub](https://img.shields.io/github/license/FirstLegoLeague/ms-messenger.svg)](https://github.com/FirstLegoLeague/ms-messenger/blob/master/LICENSE)
```
Comment (idanstark42) [inline README.md:6]: What are these?

File: appveyor.yml (hunk 1 above)
Comment (2roy999) [reply appveyor.yml]: My Webstorm claimed it's not by the schema and I change it automatically. I will revert it

File: README.md (hunk 2 above)
Comment (2roy999) [reply README.md:6]: badges, you can see a preview in the ms-correlation

**Review by idanstark42 — APPROVED** [review]

**Pull request merged**